graphql/graphiql.go

```diff
@@ -0,0 +1,95 @@
+// The MIT License (MIT)
```

Comment:
This has a different license from the rest of the repo, which might be a problem. It looks like it was taken from a different repo, wasn't it? If so, it should be vendored.

Reply:
It was taken from https://github.com/graphql-go/handler/blob/master/graphiql.go, where it is, inconveniently, private, and thus can't be vendored in.

It's also pretty much 100% boilerplate, so I wasn't really sure of the best way to handle it.

Reply:
Ok I see... It seems to be fairly independent from the rest of the code though, and in fact I don't see `ServeHTTP` called anywhere. What do you think of creating a separate repo with an MIT license that embeds this file, then vendor that repo?

Reply:
Seems like a bit of a waste of time to me. :/

Reply:
come on, it's 30 seconds at most. Anyhow, Peter says it doesn't matter so I'll close this point.